lib/droplet_kit/resources/database_resource.rb

```diff
@@ -13,7 +13,7 @@ class DatabaseResource < ResourceKit::Resource
 
       action :create_cluster, 'POST /v2/databases' do
         body { |object| DatabaseClusterMapping.representation_for(:create, object) }
-        handler(202) { |response, database| DatabaseClusterMapping.extract_into_object(database, response.body, :read) }
+        handler(201) { |response, database| DatabaseClusterMapping.extract_into_object(database, response.body, :read) }
```

Comment:
Hey @swanny85, thanks for the PR! I saw the test failure and realize it's a bit of a confusing error message, but changing,

https://github.com/digitalocean/droplet_kit/blob/3929a4b2c1cb5662cc4d1d11f16964dd1b199ad0/spec/lib/droplet_kit/resources/database_resource_spec.rb#L95

to mock a status of `201` instead of the current `202` will fix the test failure.